<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HRMTool</class>
 <widget class="QWidget" name="HRMTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,28">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_overview">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>78</height>
        </size>
       </property>
       <property name="text">
        <string>简历总览</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_tool">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>78</height>
        </size>
       </property>
       <property name="text">
        <string>简历录入</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_resume_selection">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>78</height>
        </size>
       </property>
       <property name="text">
        <string>简历初筛</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_mainPage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>78</height>
        </size>
       </property>
       <property name="text">
        <string>主页</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="ArchiveBrowser" name="page_archive_preview"/>
     <widget class="UploadResume" name="page_upload_resume"/>
     <widget class="ResumeSelection" name="page_resume_selection"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UploadResume</class>
   <extends>QWidget</extends>
   <header location="global">HRMTool/uploadresume.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArchiveBrowser</class>
   <extends>QWidget</extends>
   <header location="global">HRMTool/archivebrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResumeSelection</class>
   <extends>QWidget</extends>
   <header location="global">HRMTool/resumeselection.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
